README.md

```diff
@@ -1,2 +1,13 @@
 # comp1-software
 Software for Competition 1 of the "Not-A-Boring" Competition
+
+## Front-end (Angular) setup Guide
+1. First, open your terminal. Navigate to `/comp1-software` directory.
+2. Run `npm install -g @angular/cli`
+3. Navigate to `/OnLand/ControlLaptop/angular-UI`. 
+    If you are in the root of the 'comp1-software' folder, then just run:
+        Windows: `cd OnLand\ControlLaptop\angular-UI`
+        Linux/Mac: `cd OnLand/ControlLaptop/angular-UI`
+4. When you in the angular-UI folder, run `npm install`
+5. To build and serves the app locally, run `ng serve`. If it's build successfully, there should be a link that look like this: http://localhost:4200/. 
+6. Navigate to the link in your browser, and your should be able to see the app.
```

Comment:
I really appreciate you adding Windows and Linux notes, I tend to add Linux ones and then people on windows are like dude what is this...

Reply:
:smile: